datasets/ge/ot_list/crawler.py

```diff
@@ -0,0 +1,94 @@
+"""
+Crawl the website of the Legislative Herald of Georgia and extract
+the individuals on the Otkhozoria–Tatunashvili List.
+"""
+
+import re
+from typing import Dict
+
+from transliterate import translit
+from zavod import Context, Entity
+from zavod import helpers as h
+from lxml.html import HtmlElement
+
+TARGET = "ბრალდებული/ მსჯავრდებული"
+DEMOGRAPHICS = "დემოგრაფიული მონაცემები"
+STATUS = "სტატუსი"
+UNKNOWN = "უცნობია"
+UNUSED_FIELDS = [
+    "საქმის მდგომარეობა",
+    "დანაშაულის ჩადენის ადგილი",
+    "მოკლე ფაბულა",
+    "შენიშვნა",
+]
+DATE_FORMATS = ["%d.%m.%Y", "%Y"]
+PATROYNMIC = re.compile(r"\b(\S+)\s+ძე\s+")
+RUSSIANIZE = re.compile(r"([oe]v)i$")
+
+
+def extract_name(context: Context, person: Entity, name: str):
+    """Parse a Georgian name slightly and transliterate."""
+    patronym = None
+    m = PATROYNMIC.search(name)
+    if m is not None:
+        patronym = m.group(1)
+        name = name[:m.start()] + name[m.end():]
+        context.log.debug(f"Patroynmic: {m.group(1)}")
+    h.apply_name(person, full=name, patronymic=patronym, lang="geo")
+    transliterated = translit(name, "ka", reversed=True).title()
```

Comment:
We don't need to transliterate names here. The rest of our stack is intended to work across unicode alphabets, including the name matchers. Georgian transliteration is a complete mess, so the results of this would also be questionable at best.

Reply:
Ah ok - I was wondering if this was the case.  The hack to turn -ovi/-evi into -ov/-ev is unnecessary too I imagine?

Separating out the patronym, on the other hand, seems useful to me as Georgian names usually don't include it.